src/terraform/modules/policy-assignments/variables.tf

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
`terraform fmt`

Reply:
Not sure formatting changed but ran fmt on file.